cmd/influxd/run/config.go

```diff
@@ -306,3 +307,60 @@ func (c *Config) applyEnvOverrides(prefix string, spec reflect.Value, structKey
 	}
 	return nil
 }
+
+// Diagnostics returns a diagnostics representation of Config.
+func (c *Config) Diagnostics() (*diagnostics.Diagnostics, error) {
+	return diagnostics.RowFromMap(map[string]interface{}{
+		"reporting-disabled": c.ReportingDisabled,
+		"bind-address":       c.BindAddress,
+	}), nil
+}
+
+func (c *Config) diagnosticsClients() map[string]diagnostics.Client {
+	// Config settings that are always present.
+	m := map[string]diagnostics.Client{
+		"config": c,
```

Comment:
If we're including `c` do we need to explicitly include all the other things? I'm a bit unclear on this.